knowrob_common/prolog/plunit.pl

```diff
@@ -1,1531 +0,0 @@
-/*  $Id$
```

Comment:
this should not be part of the commit. Could you please restore the file for this pull request? This will result in merge conflict when I merge master in kinetic branch.

Reply:
Don't know how that happened -.-

Reply:
But wait. Back then you told me to remove that file... Still want that to be restored?